buildSrc/src/main/kotlin/mpp-module.gradle.kts

```diff
@@ -37,13 +45,16 @@ kotlin {
                 }
             }
         }
+        for (target in nativeTargets) {
+            targets.add(presets.getByName(target).createTarget(target))
+        }
     }
     sourceSets {
         commonTest {
             dependencies {
                 implementation(kotlin("test-common"))
                 implementation(kotlin("test-annotations-common"))
-                implementation("com.github.komputing.khex:extensions:${Versions.khex}")
+                implementation("KHex:extensions:${Versions.khex}")
```

Comment:
this breaks the build for me:
```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':sha512:jsIrTestPackageJson'.
> Could not resolve all dependencies for configuration ':sha512:jsIrTestNpm'.
   > Could not find KHex:extensions:1.1.2.
     Searched in the following locations:
       - https://maven.pkg.github.com/komputing/KHex/KHex/extensions/1.1.2/extensions-1.1.2.pom
       - https://repo.maven.apache.org/maven2/KHex/extensions/1.1.2/extensions-1.1.2.pom
     Required by:
         project :sha512

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 5s
```

Reply:
@ligi 

Oh oops, I forgot to mention this PR relies on this pr: https://github.com/komputing/KHex/pull/14
So this can't be merged until KHex is updated and has new artifacts published

Reply:
thanks! I merged the KHex PR - but still not sure how this can work afterwards - removing the `com.github.komputing` would mean it is not resolved via jitpack anymore - but not sure what service would be used to resolve the dependency then

Reply:
Sorry, that was a mistake. I published KHex to mavenLocal and accidentally commited that line

Reply:
no worries - but that might be a deal breaker for merging - seems jitpack still has problems with KMP: https://github.com/jitpack/jitpack.io/issues/3853

Reply:
@ligi yeah still only jvm for jitpack afaik. And as I was saying this PR need an new version of KHex published on github artifacts before this can be merged

Reply:
ah right - we can use github artifacts - then it might not be a blocker - released and thanked you here: https://github.com/komputing/KHex/releases/tag/1.1.3
but this still needs a change here so it consumes the dependency from github artifacts right?

Reply:
Yeah it still needs that updated dependency